packages/vega-util/src/merge.ts

```diff
@@ -0,0 +1,42 @@
+// Type guard for Array or TypedArray constructors
+function isArrayConstructor(ctor: Function): ctor is ArrayConstructor {
+  return ctor === Array || (typeof ctor === 'function' && ctor.prototype && ArrayBuffer.isView(ctor.prototype));
```

Comment:
Cheap way to check if the constructor is a typed array

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer/isView

Reply:
👍  I try not to over-comment but given isView is not a frequently used API, it seems appropriate to add here